assets/js/types/type-defs/cart.ts

```diff
@@ -190,6 +193,7 @@ export interface Cart {
 	items: Array< CartItem >;
 	itemsCount: number;
 	itemsWeight: number;
+	crossSells: Array< ProductResponseItem >;
```

Comment:
Is this true, we're storing `crossSells` in the same format as the API gives back, but `items` [get parsed into camelCase](https://github.com/woocommerce/woocommerce-blocks/blob/df63d2ae397b35242c40ea4eec4673c50eaabb15/assets/js/data/cart/actions.ts#L37) - maybe we could do the same for `crossSells`?

Reply:
I'm not sure what exactly you mean, @opr. Would you mind explaining it to me?

Reply:
Oops, never mind, I was misunderstanding how the cart got processed from the sever side, I thought the same happened to items in it, but it doesn't. This is fine.